CS410Assignment2/src/sensor/SensorApplication.java

```diff
@@ -0,0 +1,88 @@
+package sensor;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.awt.Rectangle;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.JTextField;
+import javax.swing.border.TitledBorder;
+import javax.swing.plaf.basic.BasicProgressBarUI;
+
+public class SensorApplication extends JFrame {
+	
+	
+	
+	public class BarUI extends BasicProgressBarUI {
+        private Rectangle r = new Rectangle();
+    }
+	
+	Color c;
+	JTextField textField; //
+	JLabel rvLabel;
+	String type;
+	String report;
+	String valueString;
+	double value;
+	int max;
+
+	
+	
+	
+
+	public SensorApplication() {
+		
+		setTitle("Sensor Tracker");
+		setLayout(new GridLayout(3, 1));
+		
+		SensorAdapter[] sensors = {new PressureSensorAdapter(new PressureSensor()),

```

Comment:
Using an interface here is solid, I didn't do that- it would've been better for me to have a sensor interface and for each loop to draw in my gridlayout, using generics is something that I should've done.